src/NUnitFramework/framework/Internal/TestExecutionContext.cs

```diff
@@ -167,7 +167,7 @@ public TestExecutionContext( TestExecutionContext other )
         [ThreadStatic]
         private static TestExecutionContext current;
 #elif NETCF
-        private static TestExecutionContext current;
+        private static LocalDataStoreSlot slotContext = Thread.AllocateDataSlot();
```

Comment:
What if we encapsulated all the differences in how current is accessed into a property?